utils/profile-actions.js

```diff
@@ -11,15 +22,20 @@ export function fillEmail(email) {
 export function fillProfile(
   options = {
     lastName: 'Coelho',
+    country: 'BRA',
   }
 ) {
+  const { lastName = 'Coelho', country = 'BRA' } = options
```

Comment:
Is it necessary, since we're already defining default params values for `options`?

Reply:
yep, because if you pass an object containing only the country, the lastName will be `undefined`. The default object in the params is only used if you call the function without any arguments :-/

Reply:
just to demonstrate, I created this simple function in node to show the outputs:

```js
function logArgs(options = { lastName: 'Coelho', country: 'BRA' }) {
  const { lastName = 'Coelho', country = 'BRA' } = options
  console.log(options)
  console.log({ lastName, country })
}
```

<img width="584" alt="image" src="https://user-images.githubusercontent.com/10223856/185404404-48f76d1f-4507-43bd-833c-abc5e7e460a7.png">

Reply:
nice to know! Thanks